src/Drivers/InputGPIO.h

```diff
@@ -0,0 +1,22 @@
+#ifndef CMF_INPUTGPIO_H
+#define CMF_INPUTGPIO_H
+
+#include "Drivers/Interface/InputDriver.h"
+#include "Periphery/GPIO.h"
+
+class InputGPIO : public InputDriver {
+	GENERATED_BODY(InputGPIO, InputDriver);
+
+public:
+	InputGPIO() = default;
+	InputGPIO(const std::vector<InputPinDef>& inputs, StrongObjectPtr<GPIO> gpio);
```

Comment:
Umjesto da passas gpio ptr kroz konstruktor, koristi SubclassOf<GPIO>, gettaj ga iz Application instancea po klasi, tako da je useru sto jednostavnije to koristiti kroz sto manje koda, jer ce inace user ovog drivera sam morati gettati gpio instance koji zeli